src/component/BpmnVisualization.ts

```diff
@@ -31,7 +31,8 @@ export default class BpmnVisualization {
       }
       // Instantiate and configure Graph
       const configurator = new MxGraphConfigurator(this.container);
-      this.graph = configurator.configure(options);
+      this.graph = configurator.configure();
+      configurator.configureMouseNavigationSupport(options);
```

Comment:
Why don't put this in `configurator.configure()` ?

Reply:
I guess I missed to reintegrate it after refactoring, the `configureMouseNavigationSupport` method was previously hold in `BpmnVisualization`
I will put everything within the `MxGraphConfigurator.configure`

Reply:
Done with 8346db6